src/domaindiag.F

```diff
@@ -10531,63 +10456,56 @@ subroutine scalar_flux(weps,c1,c2,mf,rf0,s   ,dum3,dum4,dum5,w3d,khv,sflux ,wavg
           if(bc_temp.eq.1)then
             ! specified theta at boundary
 
-#ifndef _B4B26R
-            !$acc loop vector collapse(2) reduction(+:sfst1,sfst2) 
-#endif
+            !$acc loop gang vector collapse(2) reduction(-:sfst1,sfst2) 
             do j=1,nj
             do i=1,ni
               sfst1 = sfst1-(viscosity/pr_num)*2.0*(savg(1)-ptc_bot)*rdz*mf(1,1,1)
```

Comment:
It appears that ptc_bot and ptc_top are not likely initialized properly on the GPU.  Is this addressed in a future PR?

Reply:
Thanks